core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.ts

```diff
@@ -51,17 +51,21 @@ interface MergeModalResult {
   templateUrl: './skills-list.component.html'
 })
 export class SkillsListComponent {
-  @Input() skillSummaries: AugmentedSkillSummary[];
-  @Input() pageNumber: number;
-  @Input() itemsPerPage: number;
-  @Input() editableTopicSummaries: CreatorTopicSummary[];
-  @Input() mergeableSkillSummaries: SkillSummary[];
-  @Input() untriagedSkillSummaries: SkillSummary[];
-  @Input() userCanDeleteSkill: boolean;
-  @Input() userCanCreateSkill: boolean;
-  @Input() skillsCategorizedByTopics: SkillsCategorizedByTopics;
+  // These properties below are initialized using Angular lifecycle hooks
+  // where we need to do non-null assertion. For more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() skillSummaries!: AugmentedSkillSummary[];
+  @Input() pageNumber!: number;
+  @Input() itemsPerPage!: number;
+  @Input() editableTopicSummaries!: CreatorTopicSummary[];
+  @Input() mergeableSkillSummaries!: SkillSummary[];
+  @Input() skillsCategorizedByTopics!: SkillsCategorizedByTopics;
+  @Input() untriagedSkillSummaries!: SkillSummary[];
+  @Input() userCanCreateSkill: boolean = true;
+  @Input() userCanDeleteSkill: boolean = true;
```

Comment:
I think you can refactor this and remove the `userCanCreateSkill` as it is not used anywhere, and for `userCanDeleteSkill` (it is used in the accompanying HTML file) I would just add `!` and keep it without any default value.

Reply:
Done!